core/Cargo.toml

```
@@ -21,7 +21,7 @@ lazy_static = "1.2"
 libsecp256k1 = { version = "0.3.1", optional = true }
 log = "0.4"
 multiaddr = { package = "parity-multiaddr", version = "0.9.2", path = "../misc/multiaddr" }
-multihash = "0.11.0"
+multihash = { version = "0.12", default-features = false, features = ["std", "arb", "derive", "sha2"] }
```

```suggestion
multihash = { version = "0.12", default-features = false, features = ["std", "derive", "sha2"] }
```

I think we need `Arbitrary` in test code only, right? If so, can this be moved to `dev-dependencies`?

---

Yes it can. At the moment it doesn't make any difference, but in future Rust versions it could. I'll move it down.